### ISSUE TITLE
Please support whitespace after the ticks

### ISSUE THREAD

**Issue by nolange** [pomdtr/vscode-markdown-kroki#5]
Hello,

your parser is really picky, it should ignore whitespaces, or rather eat additional ones in the text following the "ticks".
The vscode parser seems to do the right thing, still detecting the code blocks, but your plugin will refuse for ex. the following:

~~~~ markdown
``` mermaid
graph TD;
    A-->B;
    A-->C;
    B-->D;
    C-->D;
```

(whitespace behind `mermaid`)
```mermaid 
graph TD;
    A-->B;
    A-->C;
    B-->D;
    C-->D;
```
~~~~

Commonmark has some [Examples](https://spec.commonmark.org/0.30/#example-143) on what to expect.

**Comment by Airkro** [comment]
it should support parse markdown code meta, see: https://github.com/nice-move/markdown-code-block-meta

**Comment by pomdtr** [comment]
fixed in `v1.1.1`
